src/clarity-angular/popover/dropdown/dropdown.ts

```diff
@@ -56,6 +61,11 @@ export class Dropdown {
     constructor(public elementRef: ElementRef, @SkipSelf() @Optional() public parent: Dropdown,
                 public ifOpenService: IfOpenService) {
         this.anchor = elementRef.nativeElement;
+
+        // Subscribe and broadcast openChanges from the service
+        ifOpenService.openChange.takeUntil(this.ngUnsubscribe).subscribe(change => {
```

Comment:
I wasn't 100% sure what we decided. This is easy to change to a Subscription unsubscribe as we do other places if it was decided not to go this route. 

One thing I like about it though is that the `takeUntil` just operates on the subscription provided by the service and we don't have to assign it to a new one that later gets cleaned up 
idk if its better but it seems cleaner to me. 

```
let mySubscription = ifOpenService.openChange.subscribe(...)
```